image/memory.go

```diff
@@ -71,3 +71,10 @@ func (i *memoryImage) Inspect() (*types.ImageInspectInfo, error) {
 func (i *memoryImage) IsMultiImage() bool {
 	return false
 }
+
+// UpdatedLayerInfos returns an updated set of layer blob information which may not match the manifest.
+// The Digest field is guaranteed to be provided; Size may be -1.
+// WARNING: The list may contain duplicates, and they are semantically relevant.
+func (i *memoryImage) UpdatedLayerInfos() []types.BlobInfo {
+	return i.LayerInfos()
```

Comment:
Can’t this be just `return nil`?

Reply:
Hmm, yeah, that'd help ensure that we don't hit the `srcInfosUpdated = true` path in `copyLayers`.  Changing it.